src/rez/cli/test.py

```diff
@@ -67,3 +95,75 @@ def command(opts, parser, extra_arg_groups=None):
 
         if returncode:
             sys.exit(returncode)
+
+
+def is_dev_run(name):
+    """
+    Decides if is a dev_run, based on the package name
+    Args:
+        name: name of package
+
+    Returns: True or False
+    """
+    return not name or name == '.'
+
+
+def get_package(path):
+    """
+    Function validates pkg name and indicates if tests were run in developer mode
+    Args:
+        path: path to the package
+
+    Returns: Tuple of values (package_name, is_dev_run)
```

Comment:
Wrong return docstring. Also not sure we need this function.

Reply:
probably not needed, I'm guessing was added to make the code more testable, but  I'm pretty sure you can call directly `get_developer_package`